images/sphinx/Dockerfile

```diff
@@ -10,6 +11,8 @@ COPY ./usr/local/etc/sphinx/searchd.conf /usr/local/etc/sphinx/searchd.conf
 COPY ./usr/local/etc/sphinx/data/stops.txt /usr/local/etc/sphinx/data/stops.txt
 
 RUN chmod +x /root/startup.sh
+RUN chmod +x /root/*.sh
```

Comment:
I think this line makes the one above it redundant, doesn't it?

Reply:
Fixed